### PR TITLE
[mono] Add back OP_XOP_OVR_X_X for arm64

### LINKED ISSUE #85345: PNSE in System.Buffers.IndexOfAnyAsciiSearcher.IndexOfAnyVectorized on iOS/tvOS arm64
We're seeing a bunch of test failures on arm64 iOS/tvOS with stack traces like this one:

```
System.PlatformNotSupportedException : Operation is not supported on this platform.

Stack trace
   at System.Buffers.IndexOfAnyAsciiSearcher.IndexOfAnyVectorized[Negate,Default](Int16& searchSpace, Int32 searchSpaceLength, Vector128`1 bitmap)
   at System.Globalization.CompareInfo.IndexOfOrdinalHelper(ReadOnlySpan`1 source, ReadOnlySpan`1 target, CompareOptions options, Int32* matchLengthPtr, Boolean fromBeginning)
   at System.Globalization.CompareInfo.IcuIndexOfCore(ReadOnlySpan`1 source, ReadOnlySpan`1 target, CompareOptions options, Int32* matchLengthPtr, Boolean fromBeginning)
   at System.Globalization.CompareInfo.IndexOf(ReadOnlySpan`1 source, ReadOnlySpan`1 value, CompareOptions options)
   at System.Globalization.CompareInfo.IndexOf(String source, String value, Int32 startIndex, Int32 count, CompareOptions options)
   at System.String.IndexOf(String value, Int32 startIndex, Int32 count, StringComparison comparisonType)
   at System.String.IndexOf(String value, StringComparison comparisonType)
   at System.Tests.EnumTests.Parse_NonExistentValue_IncludedInErrorMessage(String value)
```

/cc @SamMonoRT @fanyang-mono @radekdoulik 

I _think_ it got introduced by https://github.com/dotnet/runtime/pull/85303

## Build Information
Build: https://github.com/dotnet/runtime/pull/85303
Leg Name: Build ios-arm64 Release AllSubsets_Mono

## Error Message

```json 
{ 
    "ErrorMessage": "at System.Buffers.IndexOfAnyAsciiSearcher.IndexOfAnyVectorized",
    "BuildRetry": false,
    "ErrorPattern": "",
    "ExcludeConsoleLog": false
} 
```

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[254131](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254131)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254131&view=ms.vss-test-web.build-test-results-tab&runId=4941566&resultId=108062)|dotnet/runtime#85445|
|[254123](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254123)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_Long_Success](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254123&view=ms.vss-test-web.build-test-results-tab&runId=4941490&resultId=108987)|dotnet/runtime#85443|
|[254098](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254098)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_Long_Success](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254098&view=ms.vss-test-web.build-test-results-tab&runId=4940734&resultId=109148)|dotnet/runtime#85437|
|[254102](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254102)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254102&view=ms.vss-test-web.build-test-results-tab&runId=4940684&resultId=109080)|dotnet/runtime#85438|
|[253680](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253680)|dotnet/runtime|[System.Tests.StringTests.ContainsNoMatch_StringComparison](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253680&view=ms.vss-test-web.build-test-results-tab&runId=4940436&resultId=107459)|dotnet/runtime#85310|
|[254054](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254054)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.ConstructorHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254054&view=ms.vss-test-web.build-test-results-tab&runId=4939704&resultId=108478)|dotnet/runtime#85433|
|[254082](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254082)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254082&view=ms.vss-test-web.build-test-results-tab&runId=4939880&resultId=109129)||
|[254073](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254073)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254073&view=ms.vss-test-web.build-test-results-tab&runId=4940090&resultId=109097)|dotnet/runtime#85431|
|[254043](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254043)|dotnet/runtime|[System.Xml.XmlSchemaValidatorApiTests.TCXmlSchemaValidatorMisc.XSDValidationGeneratesInvalidError_1](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254043&view=ms.vss-test-web.build-test-results-tab&runId=4939626&resultId=111775)|dotnet/runtime#84871|
|[254002](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254002)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=254002&view=ms.vss-test-web.build-test-results-tab&runId=4938918&resultId=109148)||
|[253912](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253912)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253912&view=ms.vss-test-web.build-test-results-tab&runId=4937970&resultId=107773)|dotnet/runtime#85424|
|[253464](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253464)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253464&view=ms.vss-test-web.build-test-results-tab&runId=4936636&resultId=108880)|dotnet/runtime#85404|
|[253894](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253894)|dotnet/runtime|[System.Tests.DerivedException.Exception_OverriddenToStringOnInnerException](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253894&view=ms.vss-test-web.build-test-results-tab&runId=4936806&resultId=108475)||
|[253831](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253831)|dotnet/runtime|[System.Tests.DerivedException.Exception_OverriddenToStringOnInnerException](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253831&view=ms.vss-test-web.build-test-results-tab&runId=4936246&resultId=108782)|dotnet/runtime#84793|
|[253826](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253826)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253826&view=ms.vss-test-web.build-test-results-tab&runId=4936006&resultId=108371)|dotnet/runtime#85283|
|[253841](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253841)|dotnet/runtime|[System.Data.Tests.DataTableTest5.XmlTest14](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253841&view=ms.vss-test-web.build-test-results-tab&runId=4935554&resultId=119197)||
|[253809](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253809)|dotnet/runtime|[System.IO.Tests.FileLoadExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253809&view=ms.vss-test-web.build-test-results-tab&runId=4935560&resultId=108054)|dotnet/runtime#85328|
|[253756](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253756)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253756&view=ms.vss-test-web.build-test-results-tab&runId=4934638&resultId=108647)|dotnet/runtime#83854|
|[253779](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253779)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253779&view=ms.vss-test-web.build-test-results-tab&runId=4933850&resultId=108844)||
|[253729](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253729)|dotnet/runtime|[System.Tests.DerivedException.Exception_OverriddenToStringOnInnerException](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253729&view=ms.vss-test-web.build-test-results-tab&runId=4934060&resultId=107224)|dotnet/runtime#85418|
|[253760](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253760)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253760&view=ms.vss-test-web.build-test-results-tab&runId=4933264&resultId=108182)||
|[253577](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253577)|dotnet/runtime|[System.Tests.BadImageFormatExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253577&view=ms.vss-test-web.build-test-results-tab&runId=4930762&resultId=108690)|dotnet/runtime#85410|
|[253537](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253537)|dotnet/runtime|[System.Tests.BadImageFormatExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253537&view=ms.vss-test-web.build-test-results-tab&runId=4930140&resultId=108252)|dotnet/runtime#85308|
|[253522](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253522)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253522&view=ms.vss-test-web.build-test-results-tab&runId=4929858&resultId=107722)|dotnet/runtime#85405|
|[253493](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253493)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_InvalidSurrogatePairs](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253493&view=ms.vss-test-web.build-test-results-tab&runId=4928994&resultId=108693)|dotnet/runtime#85343|
|[253456](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253456)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253456&view=ms.vss-test-web.build-test-results-tab&runId=4926974&resultId=108932)|dotnet/runtime#85402|
|[253435](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253435)|dotnet/runtime|[System.Tests.DerivedException.Exception_OverriddenToStringOnInnerException](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253435&view=ms.vss-test-web.build-test-results-tab&runId=4926052&resultId=108510)|dotnet/runtime#85254|
|[253413](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253413)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_Long_Success](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253413&view=ms.vss-test-web.build-test-results-tab&runId=4925266&resultId=108449)|dotnet/runtime#85400|
|[252698](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252698)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252698&view=ms.vss-test-web.build-test-results-tab&runId=4924074&resultId=107089)|dotnet/runtime#85283|
|[253374](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253374)|dotnet/runtime|[System.Tests.StringTests.ContainsNoMatch_StringComparison](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253374&view=ms.vss-test-web.build-test-results-tab&runId=4923834&resultId=107790)|dotnet/runtime#85399|
|[253359](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253359)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253359&view=ms.vss-test-web.build-test-results-tab&runId=4923104&resultId=107882)|dotnet/runtime#85323|
|[253317](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253317)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253317&view=ms.vss-test-web.build-test-results-tab&runId=4922286&resultId=108884)|dotnet/runtime#85328|
|[253228](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253228)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253228&view=ms.vss-test-web.build-test-results-tab&runId=4920948&resultId=108990)|dotnet/runtime#85335|
|[253181](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253181)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253181&view=ms.vss-test-web.build-test-results-tab&runId=4920832&resultId=108590)|dotnet/runtime#85143|
|[253162](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253162)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253162&view=ms.vss-test-web.build-test-results-tab&runId=4920676&resultId=108539)|dotnet/runtime#85393|
|[253171](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253171)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253171&view=ms.vss-test-web.build-test-results-tab&runId=4920370&resultId=108938)|dotnet/runtime#85310|
|[253139](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253139)|dotnet/runtime|[System.Tests.UriMethodTests.GetComponents_Advanced](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253139&view=ms.vss-test-web.build-test-results-tab&runId=4919122&resultId=108864)|dotnet/runtime#85313|
|[253113](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253113)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253113&view=ms.vss-test-web.build-test-results-tab&runId=4918822&resultId=108849)|dotnet/runtime#85308|
|[253059](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253059)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=253059&view=ms.vss-test-web.build-test-results-tab&runId=4915002&resultId=107613)|dotnet/runtime#85342|
|[252996](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252996)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252996&view=ms.vss-test-web.build-test-results-tab&runId=4913412&resultId=108145)|dotnet/runtime#85388|
|[252953](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252953)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252953&view=ms.vss-test-web.build-test-results-tab&runId=4911834&resultId=108646)|dotnet/runtime#85158|
|[252972](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252972)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252972&view=ms.vss-test-web.build-test-results-tab&runId=4912314&resultId=108485)|dotnet/runtime#85313|
|[252946](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252946)|dotnet/runtime|[System.Data.Tests.DataTableTest5.XmlTest9](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252946&view=ms.vss-test-web.build-test-results-tab&runId=4911740&resultId=114578)|dotnet/runtime#85384|
|[252941](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252941)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252941&view=ms.vss-test-web.build-test-results-tab&runId=4911700&resultId=107875)|dotnet/runtime#85384|
|[252926](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252926)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252926&view=ms.vss-test-web.build-test-results-tab&runId=4911442&resultId=108684)|dotnet/runtime#85328|
|[252922](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252922)|dotnet/runtime|[System.Reflection.Tests.TypeInfoPropertyTests.TestIsEnum](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252922&view=ms.vss-test-web.build-test-results-tab&runId=4911256&resultId=120076)||
|[252855](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252855)|dotnet/runtime|[WasmTestOnBrowser-Microsoft.Extensions.Http.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252855&view=ms.vss-test-web.build-test-results-tab&runId=4908370&resultId=184191)|dotnet/runtime#85143|
|[252877](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252877)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252877&view=ms.vss-test-web.build-test-results-tab&runId=4909438&resultId=108755)||
|[252815](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252815)|dotnet/runtime|[System.Runtime.Serialization.Xml.XsdDataContractExporterTests.ExporterApiTests.Ctor_Schemas](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252815&view=ms.vss-test-web.build-test-results-tab&runId=4907510&resultId=114572)|dotnet/runtime#85185|
|[252802](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252802)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252802&view=ms.vss-test-web.build-test-results-tab&runId=4907148&resultId=108902)|dotnet/runtime#85378|
|[252793](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252793)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252793&view=ms.vss-test-web.build-test-results-tab&runId=4907014&resultId=108409)|dotnet/runtime#84580|
|[252784](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252784)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252784&view=ms.vss-test-web.build-test-results-tab&runId=4906712&resultId=108998)|dotnet/runtime#85284|
|[252568](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252568)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252568&view=ms.vss-test-web.build-test-results-tab&runId=4904158&resultId=108750)|dotnet/runtime#85368|
|[252554](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252554)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_InvalidSurrogatePairs](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252554&view=ms.vss-test-web.build-test-results-tab&runId=4903970&resultId=108562)|dotnet/runtime#85282|
|[251958](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251958)|dotnet/runtime|[System.Tests.DuplicateWaitObjectExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251958&view=ms.vss-test-web.build-test-results-tab&runId=4903920&resultId=108158)|dotnet/runtime#85127|
|[252476](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252476)|dotnet/runtime|[System.Tests.StringTests.ContainsNoMatch_StringComparison](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252476&view=ms.vss-test-web.build-test-results-tab&runId=4903152&resultId=108995)|dotnet/runtime#85328|
|[252461](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252461)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252461&view=ms.vss-test-web.build-test-results-tab&runId=4902418&resultId=109162)|dotnet/runtime#85342|
|[252455](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252455)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.StructHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252455&view=ms.vss-test-web.build-test-results-tab&runId=4902108&resultId=108435)|dotnet/runtime#85191|
|[252149](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252149)|dotnet/runtime|[System.Tests.BadImageFormatExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252149&view=ms.vss-test-web.build-test-results-tab&runId=4900910&resultId=108738)||
|[252407](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252407)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252407&view=ms.vss-test-web.build-test-results-tab&runId=4900928&resultId=108999)|dotnet/runtime#85335|
|[252361](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252361)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252361&view=ms.vss-test-web.build-test-results-tab&runId=4899966&resultId=108818)|dotnet/runtime#85310|
|[252328](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252328)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252328&view=ms.vss-test-web.build-test-results-tab&runId=4899476&resultId=109163)|dotnet/runtime#84132|
|[252248](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252248)|dotnet/runtime|[System.Tests.UriMethodTests.MakeRelative_Invalid](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252248&view=ms.vss-test-web.build-test-results-tab&runId=4898332&resultId=107497)|dotnet/runtime#85360|
|[252256](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252256)|dotnet/runtime|[System.Xml.XmlSchemaValidatorApiTests.TCXmlSchemaValidatorMisc.XSDValidationGeneratesInvalidError_1](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252256&view=ms.vss-test-web.build-test-results-tab&runId=4897396&resultId=114820)||
|[252200](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252200)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252200&view=ms.vss-test-web.build-test-results-tab&runId=4896628&resultId=108268)|dotnet/runtime#85308|
|[252191](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252191)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252191&view=ms.vss-test-web.build-test-results-tab&runId=4896002&resultId=109165)|dotnet/runtime#85351|
|[252151](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252151)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252151&view=ms.vss-test-web.build-test-results-tab&runId=4895090&resultId=108869)|dotnet/runtime#85355|
|[252132](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252132)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252132&view=ms.vss-test-web.build-test-results-tab&runId=4894228&resultId=108322)|dotnet/runtime#85298|
|[252108](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252108)|dotnet/runtime|[System.Tests.UriMethodTests.EscapeDataString_InvalidSurrogatePairs](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252108&view=ms.vss-test-web.build-test-results-tab&runId=4892458&resultId=108822)|dotnet/runtime#85350|
|[252073](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252073)|dotnet/runtime|[System.Tests.StackTraceHiddenAttributeTests.MethodHidden_EnvironmentStackTrace](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252073&view=ms.vss-test-web.build-test-results-tab&runId=4892154&resultId=108933)|dotnet/runtime#85348|
|[252037](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252037)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=252037&view=ms.vss-test-web.build-test-results-tab&runId=4890986&resultId=106210)|dotnet/runtime#85346|
|[251733](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251733)|dotnet/runtime|[System.IO.Tests.FileLoadExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251733&view=ms.vss-test-web.build-test-results-tab&runId=4890070&resultId=108255)|dotnet/runtime#85334|
|[251910](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251910)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251910&view=ms.vss-test-web.build-test-results-tab&runId=4888526&resultId=109164)|dotnet/runtime#85282|
|[251686](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251686)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251686&view=ms.vss-test-web.build-test-results-tab&runId=4887970&resultId=108532)|dotnet/runtime#85328|
|[251855](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251855)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251855&view=ms.vss-test-web.build-test-results-tab&runId=4885926&resultId=108857)|dotnet/runtime#85343|
|[251867](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251867)|dotnet/runtime|[System.Tests.DerivedException.Exception_OverriddenToStringOnInnerException](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251867&view=ms.vss-test-web.build-test-results-tab&runId=4886470&resultId=108983)|dotnet/runtime#82045|
|[251842](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251842)|dotnet/runtime|[System.Tests.ArgumentExceptionTests.Ctor_String_String_Exception](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251842&view=ms.vss-test-web.build-test-results-tab&runId=4885796&resultId=108907)|dotnet/runtime#85342|
|[251820](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251820)|dotnet/runtime|[System.Tests.ArgumentOutOfRangeExceptionTests.Ctor_String_Object_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251820&view=ms.vss-test-web.build-test-results-tab&runId=4885244&resultId=108815)|dotnet/runtime#85143|
|[251772](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251772)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251772&view=ms.vss-test-web.build-test-results-tab&runId=4884068&resultId=108001)|dotnet/runtime#85313|
|[251680](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251680)|dotnet/runtime|[System.IO.Tests.FileNotFoundExceptionTests.ToStringTest](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251680&view=ms.vss-test-web.build-test-results-tab&runId=4880444&resultId=107790)|dotnet/runtime#82393|
|[251699](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251699)|dotnet/runtime|[System.Tests.ArgumentNullExceptionTests.Ctor_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251699&view=ms.vss-test-web.build-test-results-tab&runId=4880874&resultId=108427)|dotnet/runtime#74623|
|[251278](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251278)|dotnet/runtime|[System.Tests.ObjectDisposedExceptionTests.Ctor_String_String](https://dev.azure.com/dnceng-public/public/_build/results?buildId=251278&view=ms.vss-test-web.build-test-results-tab&runId=4877320&resultId=108790)|dotnet/runtime#85303|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|48|82|82|

### PR DESCRIPTION
**Pull request by radekdoulik**
And also add missing intrinsic attributes.

This addresses feedback from https://github.com/dotnet/runtime/pull/85303

Fixes https://github.com/dotnet/runtime/pull/85345

### CODE REVIEW

**Review by vargaz — APPROVED** [review]

File: src/mono/mono/mini/mini-llvm.c [hunk 1]
```diff
@@ -8704,12 +8710,6 @@ MONO_RESTORE_WARNING
 			values [ins->dreg] = call_intrins (ctx, id, args, "");
 			break;
 		}
-		case OP_XOP_OVR_X_X: {
-			IntrinsicId iid = (IntrinsicId) ins->inst_c0;
```
Comment (vargaz) [inline src/mono/mono/mini/mini-llvm.c:8708]: These probably can be moved outside the ifdefs.

**Comment by akoeplinger** [comment]
Thanks, this fixes the issues I saw on iOS. Please backport to release/8.0-preview4

File: src/mono/mono/mini/mini-llvm.c (hunk 1 above)
Comment (fanyang-mono) [reply src/mono/mono/mini/mini-llvm.c:8708]: Where were they before?

**Review by fanyang-mono — APPROVED** [review]

**Comment by radekdoulik** [comment]
The failures are unrelated

**Pull request merged**